release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -793,16 +844,43 @@ This dashboard provides API server metrics, such as:
 
 // TODO: Link to troubleshooting using this dashboard section once it is available
 
-[id="ocp-4-7-monitoring-etcd-alerts"]
-==== New etcd alerts
+[id="ocp-4-7-monitoring-namespace-by-pod-and-pod-total-Grafana-dashboards"]
+==== Enable Namespace (Pods) and Pod Kubernetes networking dashboards in Grafana
```

Comment:
I wondered if this title meant that administrators could enable the dashboards, or if they have been enabled by default in the release. I wondered if "Enabled" would more clearly describe this, but it doesn't resolve the potential ambiguity.

To resolve this, you could update the title to the following:

"Namespace (Pods) and Pod Kubernetes networking dashboards are enabled in Grafana"